cordova/config.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version='1.0' encoding='utf-8'?>
+<widget id="hk.goodcity.browsestaging" version="1.0.0" xmlns="http://www.w3.org/ns/widgets" xmlns:cdv="http://cordova.apache.org/ns/1.0">
+  <name>S. Browse</name>
+  <description>
+      Goodcity Browse Application
```

Comment:
Use `The GoodCity initiative is a new way to donate quality goods in Hong Kong. See www.goodcity.hk for more details.`